.github/workflows/publish.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: ./.github/actions/ci-setup
 
-      - run: pnpm build
+      - run: pnpm size-test build
```

Comment:
Since you consistently had to filter out `sdk-size-test#build`, it's _far_ easier to simply rename that script.